automatic/qtox/update.ps1

```diff
@@ -1,34 +1,72 @@
 ﻿import-module au
 
```

Comment:
We need an import of the chocolateyInstaller.psm1 file here.
i.e.
`Import-Module "$env:ChocolateyInstall\helpers\chocolateyInstaller.psm1"`